File: lab1/src/main/java/edu/miu/demo/entity/Post.java
package edu.miu.demo.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Post {
    private long id;
    private String title;
    private String content;
    private String author;

    public Post(long id, String title, String author) {
        this.id = id;
        this.title = title;
        this.author = author;
    }
}

File: lab9/src/components/Post/Post.js
import React from 'react'

export const Post = (props) => {
  return (
    <div className='Content'>
        <h3>{props.title}</h3>
        <div className='Field'>
            {props.content}
        </div>
        {props.author}
    </div>
  )
}

export default Post;